### PR TITLE
Update project page.

### LINKED ISSUE #35: Make project name links `target="_blank"`
The names of each project that are links (see screenshot for example) should open in a new tab, since they link to external project sites.
Screen Shot 2019-07-25 at 12 01 45 PM

### PR DESCRIPTION
**Pull request by anlawyer**
This PR closes #35.

Also added Oakland Art Murmur project to Active Project list, added conditional link rendering for project titles to only be a link if they have one in the data file, and updated some slack id's.

### CODE REVIEW

File: src/_includes/project.html [hunk 1]
```diff
@@ -1,24 +1,36 @@
-
 <h3>
-    <a href="{{ project.link }}">{{ project.name }}</a>
+  {% if project.link %}
+  <a target="_blank" href="{{ project.link }}">{{ project.name }}</a>
+  {% elsif project.link == nil %}
+  {{ project.name }}
```
Comment (ckingbailey) [inline src/_includes/project.html]: Project headings that aren't links are not getting the same left-padding as ones that are links
Screen Shot 2019-07-30 at 7 59 43 PM

File: src/_data/active_projects.yml [hunk 2]
```diff
@@ -1,64 +1,63 @@
...
   leader: Jess Sand
-  slack_id: 
+  slack_id: C95U7371C
   slack_channel: woaq
   description: We are creating a digital platform for West Oakland residents to collect, visualize, and act on the air quality data which affect them. We are partnered with the West Oakland Environmental Indicators Project, a community group who has collected Air Quality data from their neighborhood.
--
-  name: OUSD Budget
-  image: 
-  link: 
+- name: OUSD Budget
```
Comment (ckingbailey) [inline src/_data/active_projects.yml]: Do we want this page to be alphabetical? Projects up to WOAQ are alphabetized.

File: src/_includes/project.html (hunk 1 above)
Comment (ckingbailey) [reply src/_includes/project.html]: Screen Shot 2019-07-30 at 8 00 09 PM

File: src/_data/active_projects.yml (hunk 2 above)
Comment (anlawyer) [reply src/_data/active_projects.yml]: I dunno, do we? I think the projects up to WOAQ came from WordPress and the next three have been added after export. Thoughts?

File: src/_includes/project.html (hunk 1 above)
Comment (anlawyer) [reply src/_includes/project.html]: Boo.

**Pull request merged**